### DISCUSSION TITLE
GitHub Desktop commits with empty line to separate subject from body

### DISCUSSION THREAD

**Discussion by Zweihorn** [community/community#62656] [New to GitHub] [answered]
Apparently, good git commit message style requires to separate subject from body with a blank line.

Unfortunately, the GitHub Desktop deletes an empty line at body head of a new commit. At least this happens to me with GitHub 3.2.7 (and earlier) on macOS and I found no preferences to change this.

Please note a `git config --global commit.cleanup verbatim` did not help. Apparently, GitHub Desktop always enforces a "--cleanup=whitespace", if I am correct.

ref [git-commit - cleanup](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---cleanupltmodegt)

ref [Commit Message Guidelines](https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53)

Naturally, there would be CLI git but how to accomplish this (basic) task with GitHub Desktop?

**Comment by Zweihorn** [comment]
THX @MahsaBakhtiari and I appreciate this.

> hey Zweihorn, I hope this helps you:
> 
>     1. **Add a Dummy Line**: 

Good proposal. However, the blank space would not meet the requirement of an "empty line" apparently.

>     2. **Use Git CLI**: 

I was aware of this. Nevertheless worth to be included here for the benefit of a wider community.

Unfortunately, your kind and able answer does not provide a solution for the a.m. basic task by help of the GitHub Desktop App.

**Comment by queenofcorgis** [comment]
Hi there @Zweihorn - would you mind providing screenshots of the issue please?

**Reply by Zweihorn** [reply]
Yep, have a look at: https://github.com/desktop/desktop/issues/17175

**Comment by Zweihorn** [comment]
Another solution outside the GitHub Desktop would be:

**3. Use CLI or Git Bash:** 
For example `git commit -m $'message subject\n\nmessage body'` by making use of the `$'...'` feature of Bash.

**Comment by Zweihorn** [comment, marked as answer]
Oh silly me and this was a most embarrassingly misjudgement from my side. Nevertheless a good learning experience.

**There is no fail with the GitHub Desktop App and the empty separation line is provided as default.**

A basic `git log` would have shown the technical details and would have helped me to avoid causing unnecessary effort.

**However, I would like to thank for the able and kind advice of this great community.** Please accept my sincere apologies for any inconvenience.

🌻
